server/chat-plugins/room-faqs.ts

```diff
@@ -133,6 +140,46 @@ export const commands: ChatCommands = {
 	],
 };
 
+export const pages: PageTable = {
+	roomfaqs(args, user) {
+		const room = this.requireRoom();
+		this.title = `[Room FAQs]`;
+		// allow it for users if they can access the room
+		if (!user.inRooms.has(room.roomid) && room.settings.isPrivate && !user.isStaff) {
+			return this.errorReply(`Access denied.`);
+		}
+		let buf = `<div class="pad"><button style="float:right;" class="button" name="send" value="/join view-roomfaqs-${room.roomid}"><i class="fa fa-refresh"></i> Refresh</button>`;
+		if (!roomFaqs[room.roomid]) {
+			return `${buf}<h2>This room has no FAQs.</h2></div>`;
+		}
+
+		buf += `<h2>FAQs for ${room.title}:</h2>`;
+		const keys = Object.keys(roomFaqs[room.roomid]).filter(
+			val => !getAlias(room.roomid, val)
+		).sort((a, b) => a.localeCompare(b));
+		for (const key of keys) {
+			const topic = roomFaqs[room.roomid][key];
+			buf += `<div class="infobox">`;
+			buf += `<h3>${key}</h3>`;
+			buf += `<hr />`;
+			buf += Chat.formatText(topic, true).replace(/\n/g, '<br />');
+			const aliases = Object.keys(roomFaqs[room.roomid]).filter(val => getAlias(room.roomid, val) === key);
+			if (aliases.length) {
+				buf += `<hr /><strong>Aliases:</strong> ${aliases.join(', ')}`;
+			}
+			if (user.can('ban', null, room)) {
+				const src = Utils.escapeHTML(topic).replace(/\n/g, `<br />`);
+				buf += `<hr /><details><summary>Raw text</summary>`;
```

Comment:
```suggestion
				buf += `<hr /><details><summary>Source:</summary>`;
```
consistency with previous

Reply:
These are the same

Reply:
Also, I don't want to put the colon in a details readmore because it just looks like a colon leading into nothing on the page

Reply:
The "read more" should be immediately after the colon if it's `class="readmore"`

Reply:
It’s not a read more, just a drop down code block